graphviz.spec

```diff
@@ -223,8 +252,8 @@ BuildRequires: python-devel
 This package provides the Python extension for %{name}.
 
 %files -n python-graphviz
-%{_libdir}/graphviz/python
-%py_platsitedir/*
+%{_libdir}/graphviz/python3/*
+%{_libdir}/python3.4/site-packages/*
```

Comment:
Let's not hardcode 3.4, this is already no longer the case on Cooker...
%{_libdir}/python3*/site-packages/* or better yet %{py3_platsitedir}/*

Reply:
Yep changed